jarvis_files/jarvis_cmd/src/jarvis/build.py

```diff
@@ -249,12 +249,11 @@ def launch_loc(ctx, opts, ssh):
 def launch_auton(ctx, opts, ssh):
     build_deps(ctx)
     
-    opt = ['']
     lcm_echo = "lcm_tools/echo"
     l = 'True'
```

Comment:
!

Reply:
I wasn't using that for anything. I was trying to figure out how to set opts to null but kind of gave up

Reply:
This comment was addressing the l variable